software/src/initFunctions.h

```diff
@@ -251,7 +256,10 @@ void initStrip(){
   }
   else{
     //TODO
-    //busHandler = new AnimationHandlerBus()
+
+    //get strip length:
+    int length = atoi(strip_length);
+    busHandler = new AnimationHandlerBus(length, Silo, varSiloChanged, micHandler);
```

Comment:
This is ugly and wrong

Reply:
And by far the worst thing i've ever seen in my entire coding career